docs/docusaurus.config.js

```diff
@@ -43,7 +39,11 @@ module.exports = {
     },
     prism: {
       theme: Object.assign(require('prism-react-renderer/themes/github'), prismCustomColors)
-    }
+    },
+    gtag: {
+      trackingID: 'G-BNB79447GR',
```

Comment:
Why is the tracing ID different now? Bear in mind I've got no idea how this stuff works

Reply:
Earlier it was not a GA4 id, it was an UA id. UA is deprecated now and so its all the docusaurus modules around it. Thus now we have to setup the GA4 analytics.